apps/gallery/test/marionette/apps/fakeactivitycaller/manifest.webapp

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Fake Gallery Activity Caller",
+  "description": "Fake Gallery activity caller app",
+  "type": "certified",
```

Comment:
Try to use the lowest privilege necessary. It should work as type:privileged, I think, even if you use device storage to read a file for the open test.